app/helpers/puppet_related_helper.rb

```diff
@@ -68,7 +68,7 @@ def interesting_puppetclasses(obj)
   end
 
   def extraction_warning
-    "being extracted to the foreman_puppet plugin and will be removed from core in 3.0"
+    _("being extracted to the foreman_puppet plugin and will be removed from core in 3.0")
```

Comment:
I don't think this is right because you end up with double translations. It's included in translatable strings.

Reply:
Also, although the people translating this are smart enough to not translate `foreman_puppet`, I'd recommend to exclude this from the translated string via `%s` or the default way for extracting variables in Foreman.